csrankings-z.csv

```diff
@@ -44,8 +44,8 @@ Zeyu Mi,Shanghai Jiao Tong University,https://ipads.se.sjtu.edu.cn/pub/members/z
 Zeyun Yu,University of Wisconsin - Milwaukee,https://uwm.edu/engineering/people/yu-ph-d-zeyun,9kT9SaIAAAAJ
 Zhan Qin,Zhejiang University,https://person.zju.edu.cn/qinzhan,5fa4lOQAAAAJ
 Zhang Xiong,Beihang University,http://scse.buaa.edu.cn/info/1078/2637.htm,NOSCHOLARPAGE
-Zhangyang (Atlas) Wang,Texas A&M University,http://www.atlaswang.com,pxFyKAIAAAAJ
-Zhangyang Wang,Texas A&M University,http://www.atlaswang.com,pxFyKAIAAAAJ
+Zhangyang (Atlas) Wang,University of Texas at Austin,http://www.atlaswang.com,pxFyKAIAAAAJ

```

Comment:
Not listed as affiliated faculty (https://www.cs.utexas.edu/people); please remove this entry and the one below.